pkg/plugin/view/vulnerabilities/sort.go

```diff
@@ -0,0 +1,27 @@
+package vulnerabilities
+
+import (
+	starboard "github.com/aquasecurity/starboard/pkg/apis/aquasecurity/v1alpha1"
+)
+
+type VulnerabilityItems []starboard.VulnerabilityItem
+
+func (s VulnerabilityItems) Len() int { return len(s) }
+
+func (s VulnerabilityItems) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
+
+// BySeverity implements sort.Interface by providing Less and using the Len and
+// Swap methods of the embedded VulnerabilityItems value.
+type BySeverity struct{ VulnerabilityItems }
+
+var severityOrder = map[starboard.Severity]int{
+	starboard.SeverityCritical: 5,
+	starboard.SeverityHigh:     4,
+	starboard.SeverityMedium:   3,
+	starboard.SeverityLow:      2,
+	starboard.SeverityUnknown:  1,
+}
+
+func (s BySeverity) Less(i, j int) bool {
+	return severityOrder[s.VulnerabilityItems[i].Severity] > severityOrder[s.VulnerabilityItems[j].Severity]
```

Comment:
Am I going mad, or is this greater than? I think it would be better to keep this as less, and either reverse sort or change the numbering if it needs to be the other way around

Reply:
You're absolutely right. The ordering was mad :) I've fixed it!